pkgs/games/quakespasm/vulkan.nix

```
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vkquake";
-  majorVersion = "1.01";
-  version = "${majorVersion}.0";
+  majorVersion = "1.02";
+  version = "${majorVersion}.1";
```

i don't  see majorVersion being used anywhere, do you mind just combining the two?
```suggestion
  version = "1.02.1";
```

---

@jonringer Done. This was a copy-paste from `pkgs/games/quakespasm/default.nix`